packages/core/src/layout/Sidebar/Bar.tsx

```diff
@@ -44,6 +44,9 @@ const useStyles = makeStyles<BackstageTheme>(theme => ({
       easing: theme.transitions.easing.sharp,
       duration: theme.transitions.duration.shortest,
     }),
+    '& > *': {
```

Comment:
@Rugvip before you say anything, I feel it's actually pretty clean that the one that declares the flex container also decides the flex behavior of its immediate children :)

Reply:
Since it's all self contained in this component, let's go with this and refactor if we end up needing something different